.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
+  <profile version="1.0">
+    <option name="myName" value="Project Default" />
+    <inspection_tool class="Eslint" enabled="true" level="WARNING" enabled_by_default="true" />
+  </profile>
+</component>
```

Comment:
Tried adding a new line but apparently it was reset before committing, I think since this is an auto-generated IDE settings file we might not be able to really modify it.